skiko/src/androidMain/kotlin/org/jetbrains/skiko/Renderer.android.kt

```diff
@@ -123,8 +135,8 @@ private class SkikoSurfaceRender(private val layer: SkiaLayer) : GLSurfaceView.R
             canvas?.drawPicture(it.instance)
             Unit
         }
-
         context?.flush()
+        manager.onFrameCompleted()
```

Comment:
Thanks!

This is better, but we still need to figure out how to wait for swapBuffers, otherwise we still render as fast as possible, consuming the battery.

Reply:
Why do you think that context flush doesn't swap buffers?